### ISSUE TITLE
 (Dark Theme) Tab highlight color is not correct.

### ISSUE THREAD

**Issue by kavimuru** [Expensify/App#13256]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

## Action Performed:

1. Open the app on web.
2. Click on the search icon on LHN.
3. Use Escape key to close the page.
4. Press Tab multiple times until first item on the LHN is highlighted.

## Expected Result:
The Highlight outline should match the dark theme.

## Actual Result:
The highlight outline is blue which is part of light theme and should match the dark theme

## Workaround:
unknown

## Platform:

Where is this issue occurring?

- Web
- Desktop App

**Version Number:** 1.2.35-0
**Reproducible in staging?:** y
**Reproducible in production?:** y
**Email or phone of affected tester (no customers):** 
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Notes/Photos/Videos:** 
![image](https://user-images.githubusercontent.com/43996225/205190478-06a80fe1-44cb-47a8-ab94-8617d29e63c3.png)

https://user-images.githubusercontent.com/43996225/205190497-0d39c332-6daf-4571-b8be-e6d128c2c281.mp4

**Expensify/Expensify Issue URL:** 
**Issue reported by:** @parasharrajat 
**Slack conversation:** https://expensify.slack.com/archives/C049HHMV9SM/p1669646862849829

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by dhairyasenjaliya** [comment]
**Proposal**

- We need to change the color on the below file to `focus-visible` & `focus[data-focusvisible-polyfill]` property `box-shadow` to `greenHover: '#00C271',` 



- This `greenHover` color looks neat & clean and but might we need to confirm the color with the design team

https://github.com/Expensify/App/blob/a8e3cb5ecab2ed91528dbe3fb559cc215ac71349/web/index.html#L35-L42

**Result**
<img width="892" alt="Screenshot 2022-12-02 at 11 49 29 AM" src="https://user-images.githubusercontent.com/47522946/205228369-1bdd497f-e907-4c89-8044-8b896bbcf2f7.png">

**Comment by sketchydroide** [comment]
picking this up myself

**Comment by parasharrajat** [comment]
I would suggest discussing this first with Design team to decide if this is really needed. I had doubt on this as I posted on slack. 
> Note: Please discuss this issue if we really need to change the outline color.

**Comment by sketchydroide** [comment]
@Expensify/design any input here?

**Comment by michaelhaxhiu** [comment]
Let's make sure the Expected Result section of the GH is more specific, as right now it's open ended as-is:

>## Expected Result:
>The Highlight outline should match the dark theme.

**Comment by parasharrajat** [comment]
> What would matching the dark theme actually mean?

This is where the design team kicks in.

**Comment by michaelhaxhiu** [comment]
Understood. I'm not going to export this job to Upwork until we define/agree on the Expected Result.

https://expensify.slack.com/archives/C049HHMV9SM/p1669986622516479?thread_ts=1669646862.849829&cid=C049HHMV9SM

**Comment by shawnborton** [comment]
Cool, commenting over in Slack.

**Comment by sketchydroide** [comment]
seem like @shawnborton has decided on a blue color, the one from the links on the app, which I think it's still different from this one. So this GH still makes some sense

**Comment by michaelhaxhiu** [comment]
Can we delineate which blue we have now, vs the blue we want to use? I want to make sure the expected solution is accurate before we work on this further.

Also - @sketchydroide do you agree this is still `external`.

**Comment by michaelhaxhiu** [comment]
While this is externally resolvable, @sketchydroide is electing to work on it and self assigned for that reason. C+ is assigned for the PR review.

**Comment by michaelhaxhiu** [comment]
quick / stupid question - do we have light mode anymore? I don't see it in my preferences/settings. I don't miss it yet 😂 but just asking for sake up updating GH

**Comment by michaelhaxhiu** [comment]
Thanks! I missed that while I was OOO

**Comment by sketchydroide** [comment]
I created the PR

**Comment by shawnborton** [comment]
Should we follow up with a PR to round the corners, and particularly, match the roundness of buttons?

**Comment by sketchydroide** [comment]
the PR has been merged

**Comment by michaelhaxhiu** [comment]
Noice

**Comment by sketchydroide** [comment]
maybe it will go in the next release 🤞🏼

**Comment by michaelhaxhiu** [comment]
kewl

**Comment by michaelhaxhiu** [comment]
nice work moving fast :)

**Comment by sketchydroide** [comment]
this is in prod,  Ithink we can close it, unless we need to pay something, @michaelhaxhiu

**Comment by michaelhaxhiu** [comment]
>Issue reported by: @parasharrajat

$250 for rajat catching & reporting this

**Comment by michaelhaxhiu** [comment]
Accepted, waiting for you to confirm next. Drop me a comment here when you do.

**Comment by michaelhaxhiu** [comment]
@aimane-chnaif yep, don't worry! :) I will dispatch you payment in 2 days via https://www.upwork.com/jobs/~012a0e934343a83aee

Invited you to that job ^

**Comment by michaelhaxhiu** [comment]
Cool, hired!

**Comment by sketchydroide** [comment]
I don't think there is an offending PR here, it's jus that the UI changes made this apparent, maybe it was forgotten, IDK

**Comment by michaelhaxhiu** [comment]
Agreed, this isn't a bug or error in our process. Checking the final box.
